### DISCUSSION TITLE
How doo I remove server from web gui

### DISCUSSION THREAD

**Discussion by simplecoin-eu** [gravitational/teleport#11842] [Q&A] [answered]
I would like to remove teleport (self) from servers, which should be obvious but I cannot figure it out

![image](https://user-images.githubusercontent.com/60618481/162506759-c1b644b6-a013-4581-8b96-26c2744eb8c4.png)

There is only add and connect, no edit nor remove. As i see teleport is still work in progress, editing yaml would be feasible at this moment, but cannot find this either...

**Comment by webvictim** [comment, marked as answer]
https://github.com/gravitational/teleport/discussions/9920

**Reply by simplecoin-eu** [reply]
that's not it, I want vm-teleport (localhost) removed from server list but i want to use it. I think it can be security issue. Teleport server itself should be highly secured so there is no point opening another door to it. I don't want to uninstall it completely.

**Reply by webvictim** [reply]
Edit the `/etc/teleport.yaml` file on that server and set:
```
ssh_service:
  enabled: false
```

Restart Teleport and the server will disappear.
